Paywall script lazy-loads and renders correctly

cnasc opened the pull request:
# Description

<!--
Please include a summary of the change and which issue is fixed -include its number-. It's important that PRs connect to an existing issue, and we'll review this PR in part based on the content of that issue. Please also include relevant motivation and context.
-->

This PR hooks up the paywall communication a little more. Now, the `loadCheckoutModal` function is available, and it will load the iframe on the first click, and on subsequent clicks simply show the iframe that is already there.

This PR also adds some CSS and sets up the `NewDemoContent` page to use the new script when a query param is passed, which will allow us to more easily test this locally.

<img width="1514" alt="image" src="https://user-images.githubusercontent.com/9300702/76234114-a1f58b80-61ff-11ea-9615-24c118e00903.png">


# Checklist:

- [x] 1 PR, 1 purpose: my Pull Request applies to a single purpose
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have updated the docs to reflect my changes if applicable
- [x] I have added tests (and stories for frontend components) that prove my fix is effective or that my feature works
- [x] I have performed a self-review of my own code
- [x] If my code involves visual changes, I am adding applicable screenshots to this thread

<!--
PS: [Read how to write the perfect pull request](https://blog.github.com/2015-01-21-how-to-write-the-perfect-pull-request/)
-->

julien51 commented on paywall/src/components/content/NewDemoContent.tsx:
```diff
@@ -51,6 +51,7 @@ export default function NewDemoContent() {
     url.searchParams.get('unlockUserAccounts') === 'true'
   const useMetadataInputs: boolean =
     url.searchParams.get('metadataInputs') === 'true'
+  const useNewCheckout: boolean = url.searchParams.get('newCheckout') === 'true'
```
Thanks! That's going to be useful!

cnasc commented:
Removed the Greyout component because it was causing some issues, replaced it with a transparent background for the time being.

<img width="1616" alt="image" src="https://user-images.githubusercontent.com/9300702/76240409-8f804f80-6209-11ea-9a3d-ae12b2e428de.png">
